### DISCUSSION TITLE
Reloads entire clip attribute

### DISCUSSION THREAD

**Discussion by LunaWurx** [nosoop/SM-TFCustomWeaponsX#55] [Q&A]
I have created probably close to 100 weapons with this, but for the life of me I cannot find an attribute that makes a weapon reload it's entire clip at once, like how the Force of nature, soda popper, smg, syringe guns, and the shortstop do. Does such an attribute exist?

**Comment by nosoop** [comment]
It's not built into the game, but there is the [reload full clip at once](https://github.com/nosoop/SM-TFCustomAttributeStarterPack/wiki/Custom-Attribute-List#reload-full-clip-at-once) CAF attribute.
